src/common/strvararg.cpp

```diff
@@ -703,52 +703,55 @@ wxFormatString::ArgumentType DoGetArgumentType(const CharType *format,
         return wxFormatString::Arg_Unused;
     }
 
-    wxCHECK_MSG( parser.pspec[n-1] != NULL, wxFormatString::Arg_Unknown,
-                 "requested argument not found - invalid format string?" );
-
-    switch ( parser.pspec[n-1]->m_type )
+    if (0<n)
```

Comment:
inside this function you cannot be sure that n is always positive, because the format string might not contain arguments, and parser.nargs can become zero.